storages/merger/list.go

```diff
@@ -0,0 +1,134 @@
+package merger
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+
+	"github.com/allegro/akubra/log"
+	"github.com/allegro/akubra/storages/merger/s3datatypes"
+	"github.com/allegro/akubra/transport"
+)
+
+// MergeListResponses unifies responses from multiple backends
+func MergeListResponses(successes []transport.ResErrTuple) (resp *http.Response, err error) {
+	if len(successes) == 0 {
+		err = fmt.Errorf("No successful responses")
+		return
+	}
+	oContainer := objectsContainer{
+		list: make([]fmt.Stringer, 0),
+		set:  make(map[string]struct{}),
+	}
+	pContainer := objectsContainer{
```

Comment:
what `o` and `p` are? I guess that`o` means `objects`, if so, then maybe a better name would be content container. Same with `p` - `prefixesContainer`?